screens/ChangePasswordScreen.js

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from "react";
+import * as Yup from "yup";
+import { SafeAreaView, StyleSheet, Text } from "react-native";
+
+import AppFormField from "../components/AppFormField";
+import Form from "../components/Form";
+import SubmitButton from "../components/SubmitButton";
+import ErrorMessage from "../components/ErrorMessage";
+import SuccessMessage from "../components/SuccessMessage";
+import runCrypto from "../auth/crypto-hashing";
+
+const validationSchema = Yup.object().shape({
+  currentPassword: Yup.string().required().min(8).label("Current Password"),
+  newPassword: Yup.string().required().min(8).label("New Password"),
+  confirmNewPassword: Yup.string().required().min(8).label("Confirm Password"),
+});
+
+export default function ChangePasswordScreen() {
+  // Temporary user data
+  const [user, setUser] = useState({
+    id: 4,
+    name: "john",
+    password:
+      "5e884898da28047151d0e56f8dc6292773603d0d6aabbdd62a11ef721d1542d8",
+  });
+  const [errorVisible, setErrorVisible] = useState(false);
+  const [error, setError] = useState(false);
+  const [successVisible, setSuccessVisible] = useState(false);
+
+  // Reset errors upon page render
+  useEffect(() => {
+    setSuccessVisible(false);
+    setErrorVisible(false);
+  }, []);
+
+  // reset form to initial values
+  const resetForm = (values) => {
+    values["newPassword"] = "";
+    values["currentPassword"] = "";
+    values["confirmNewPassword"] = "";
+  };
+
+  const handlePasswordChange = async (values) => {
+    console.log("top level");
+    setErrorVisible(false);
+    console.log("top level");
+    // Show error if new passwords do not match
+    if (values["newPassword"] !== values["confirmNewPassword"]) {
```

Comment:
When I run this locally, I hit an error because `values` is undefined in this function. Is something missing?

Reply:
Changed something during tests, forgot to put it back! I changed it back locally and it fixed the error.